### DISCUSSION TITLE
How can I clear my workflow history?

### DISCUSSION THREAD

**Discussion by lazo23** [imixs/imixs-documents#24] [Q&A] [answered]
Hello @rsoika,

I am currently testing the workflows that i created in imixs-office-workflow. However the problem is, that the workflows are not deleted after they are done. Now there are so many under the field "my tasks" that I lose track of the new workflows that I create in imixs-office-workflow. How can I clear the history of workflows? Is there any way to do this? And why arent they automatically deleted after I am done with them?

For example:
![image](https://user-images.githubusercontent.com/103078441/219398903-316f569e-e026-42ca-b904-dd578cb68208.png)

Here as you can see the first workflow is currently created and yet not forwarded to the next role. The second workflow is done, but is still under the field "my tasks" although its done.

Greetings,
lazo

**Comment by rsoika** [comment, marked as answer]
Hi  @lazo23 
A process management system should never delete a process. This is to ensure the "auditing acceptability" (in German 'Revisionssicherheit) of a business process. 

But yes you are right, a completed task should no longer appear in the task list or under "My Tasks". To do this, you can move a task, when it is completed, into the 'Archive'. 

![image](https://user-images.githubusercontent.com/1339124/219662413-bef9e10b-05b8-4134-92b8-b0b1754b8100.png)

To achieve this, all you have to do is setting the workitem type to "workitemarchive" in your process model. You can find this setting in the task properties:

![image](https://user-images.githubusercontent.com/1339124/219663009-5e4b4f67-f2cc-452c-bf3f-eedf5f3b01fa.png)

That's it.

**Reply by lazo23** [reply]
Hello @rsoika,

It works, thank you!
